examples/example-graph-events.html

```diff
@@ -113,6 +113,22 @@ <h1>Example 4 - Events</h1>
       console.log('Event: edge-click', event);
       output.innerHTML = `Edge with id <b>${event.edge.data.id}</b> clicked!`;
     });
+
+    orb.events.on(Orb.OrbEventType.NODE_RIGHT_CLICK, (data) => {
+      data.event.preventDefault();
+      console.log("Node Right Clicked", data);
+    })
+
+    orb.events.on(Orb.OrbEventType.EDGE_RIGHT_CLICK, (data) => {
+      data.event.preventDefault();
+      console.log("Edge Right Clicked", data);
+    })
+
+    orb.events.on(Orb.OrbEventType.CANVAS_RIGHT_CLICK, (data) => {
+      data.event.preventDefault();
+      console.log("Canvas Right Clicked", data);
```

Comment:
```suggestion
      console.log("Event: mouse-right-click (canvas)", data);
      if (!data.subject) {
        output.innerHTML = `Canvas right clicked!`
      }
```

I would suggest adding these so the user can see the event message on the UI element.